### PR TITLE
Upgrade from gopkg.in/v2/yaml to gopkg.in/yaml.v2 everywhere

### PR DESCRIPTION
**Pull request by ghodss**
gopkg.in has (suddenly?) stopped supporting the URL format gopkg.in/v2/yaml, so
switch everything over to gopkg.in/yaml.v2. ghodss/yaml has already been
upgraded, so upgrade to the version that uses gopkg.in/yaml.v2 as well.

### CODE REVIEW

File: pkg/api/validation/schema.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ import (
 
 	"github.com/emicklei/go-restful/swagger"
 	"github.com/golang/glog"
-	"gopkg.in/v2/yaml"
+	"gopkg.in/yaml.v2"
```
Comment (lavalamp) [inline pkg/api/validation/schema.go]: can you switch to

```
yaml "gopkg.in/yaml.v2"
```

It makes some tools happier.

**Comment by lavalamp** [comment]
LGTM

File: pkg/api/validation/schema.go (hunk 1 above)
Comment (lavalamp) [inline pkg/api/validation/schema.go]: @thockin I seem to recall you wanting this change in a few places but I can't recall why?

Comment (thockin) [inline pkg/api/validation/schema.go]: Not me, sorry.

**Pull request merged**